doc/YAP/SyncSequence.puml

```diff
@@ -0,0 +1,42 @@
+@startuml Synchronization Sequence YAPServer
+Title Synchronization Sequence
+autoactivate on
+
+participant Server_1
+participant Server_2
+
+note across: Both Servers are out of Sync
+== SYNC of Server_1 ==
+
+Server_1 -> Server_2 : SYNC

```

Comment:
But if Server_1 is sending periodically with a fixed time period the SYNC Command, would it not be synchronous?
Or it would be asynchronous from the point of view of Server_2?

Reply:
Synchronous means the server gets the response with the return of the send routine. But this is not the case. It might happen there won't be a response at all.